### PR TITLE
Updated docsy theme to latest and fixed all styling issues

### PR DESCRIPTION
**Pull request by thebe14**
# Summary

- Updated the docsy theme to latest version
- Reduced font size to make the documentation usable at 100% zoom level
- Added support for tabbed content on pages (e.g. for installation steps that are different on Linux and Windows)
- Added copy to clipboard to code blocks
- Fixed last updated section to show user's name (with a mailto link if email available)
- Fixed double hr at bottom of pages without child pages
- Added icons to top menu
- Added repo stats to footer
- Fixed landing page feature blocks

### CODE REVIEW

**Comment by thebe14** [comment]
Running a HTML linter on a Hugo project is useless. The HTML files are just fragments, heavily sprinkled with Go blocks. The linter starts with insisting the <!doctype> tag is missing before it accepts any other tag, then just chokes on these files. Please turn it off, on this project its "insights" are garbage.

File: config.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ contentDir = "content/en"
 defaultContentLanguage = "en"
 defaultContentLanguageInSubdir = false
 # Useful when translating.
-enableMissingTranslationPlaceholders = false
+enableMissingTranslationPlaceholders = true
```
Comment (gwarf) [inline config.toml:17]: What's the reason behind this change?

File: config.toml [hunk 2]
```diff
@@ -28,17 +28,17 @@ pygmentsUseClassic = false
 pygmentsStyle = "tango"
 #pygmentsStyle = "monokailight"
 
-# First one is picked as the Twitter card image if not set on page.
-#images = ["images/project-illustration.png"]
+ # First one is picked as the Twitter card image if not set on page.
+ #images = ["images/project-illustration.png"]
```
Comment (gwarf) [inline config.toml:32]: Seems a typo/undeeded

File: config.toml [hunk 3]
```diff
@@ -28,17 +28,17 @@ pygmentsUseClassic = false
...
 
 # Configure how URLs look like per section.
 [permalinks]
 blog = "/:section/:year/:month/:day/:slug/"
 
 [outputs]
-# home = [ "HTML", "RSS", "JSON"]
 home = [ "HTML" ]
 page = [ "HTML" ]
+#section = [ "HTML", "RSS", "print"]
```
Comment (gwarf) [inline config.toml:41]:
```suggestion
# section = [ "HTML", "RSS", "print"]
```

File: config.toml [hunk 4]
```diff
@@ -61,21 +61,35 @@ anchor = "smart"
...
-[languages.en]
-title = "EGI Docs"
-description = "EGI documentation"
-languageName ="English"
-# Weight used for sorting.
-weight = 1
+  [languages.en]
+  title = "EGI Docs"
+  description = "EGI documentation"
+  languageName ="English"
```
Comment (gwarf) [inline config.toml:67]:
```suggestion
  languageName = "English"
```

File: config.toml [hunk 5]
```diff
@@ -61,21 +61,35 @@ anchor = "smart"
...
 
 # Everything below this are Site Params
 
 [params]
-copyright = "EGI Foundation"
+copyright = "EGI Foundation."
 privacy_policy = "https://www.egi.eu/privacy-policy/"
 
+custom_css = ["css/shortcodes/tabbed-panex.css", "css/tags.css", "css/egi.css"]
+custom_js = ["js/clipboard.min.js", "js/featherlight.min.js", "js/highlight.pack.js", "js/egi.js", "https://buttons.github.io/buttons.js"]
```
Comment (gwarf) [inline config.toml:88]: Can you download and include buttons.js instead of relying on an external an uncontrolled source?

File: config.toml [hunk 6]
```diff
@@ -116,46 +135,47 @@ enable = true
...
+      url = "https://community.egi.eu"
+      icon = "fa fa-comments"
+      desc = "Discussion and help from your fellow users"
+  [[params.links.user]]
+      name ="Twitter"
+      url = "https://twitter.com/EGI_eInfra"
+      icon = "fab fa-twitter"
+      desc = "Follow us on Twitter to get the latest news!"
+# Developer relevant links. These will show up on right side of footer and in the community page if you have one.
+  [[params.links.developer]]
```
Comment (gwarf) [inline config.toml:157]:
```suggestion
  # Developer relevant links. These will show up on right side of footer and in the community page if you have one.
  [[params.links.developer]]
```

File: config.toml [hunk 7]
```diff
@@ -116,46 +135,47 @@ enable = true
...
-#  url = "https://example.org/mail"
-#  icon = "fa fa-envelope"
-#  desc = "Discuss development issues around the project"
-# XXX Maybe useful to add back this somewhere
-# [[menu.shortcuts]]
-# name = "<i class='fa fa-cloud-download'></i> <label>Download</label>"
-# url = "https://github.com/EGI-Federation/EGI-Federation.github.io/archive/main.zip"
-# weight = 11
+# End user relevant links. These will show up on left/right side of footer and in the community page if you have one.
+  [[params.links.user]]
```
Comment (gwarf) [inline config.toml:146]:
```suggestion
  # End user relevant links. These will show up on left/right side of footer and in the community page if you have one.
  [[params.links.user]]
```

File: content/en/about/contributing/syntax/_index.md [hunk 8]
```diff
@@ -22,6 +22,42 @@ For references The following shortcodes are available:
 - [Hugo's shortcodes](https://gohugo.io/content-management/shortcodes/)
 - [Docsy theme shortcodes](https://www.docsy.dev/docs/adding-content/shortcodes/)
 
+### Adding chapters
+
+To create new chapters in the documentation (visible in left-side navigation
+tree), you have to create new folders under `/content/<language>` and add a file
+`_index.md` to that folder. The beginning of the markdown file contains a
+[front matter](https://gohugo.io/content-management/front-matter/) in YAML, JSON
+or TOML, holding the metadata of the page:
```
Comment (gwarf) [inline content/en/about/contributing/syntax/_index.md:31]: Let's only suggest YALM there, we don't want to mix front header types

File: content/en/about/contributing/syntax/_index.md [hunk 9]
```diff
@@ -22,6 +22,42 @@ For references The following shortcodes are available:
 - [Hugo's shortcodes](https://gohugo.io/content-management/shortcodes/)
 - [Docsy theme shortcodes](https://www.docsy.dev/docs/adding-content/shortcodes/)
 
+### Adding chapters
```
Comment (gwarf) [inline content/en/about/contributing/syntax/_index.md:25]: I'm not sure how it is for other people, I'm a bit unsure about using the wording chapter here, to me it feels that's really making more sense for books than on the web. Section would sound better to me.

File: content/en/about/contributing/syntax/_index.md [hunk 10]
```diff
@@ -85,33 +122,33 @@ Will render as:
...
 The supported languages are dependant on the syntax highlighter, which depends
 itself on the mardkown parser.
 
 {{% alert title="Note" color="info" %}} For [Hugo](https://gohugo.io/) the
 [goldmark parser](https://github.com/yuin/goldmark) is used and it relies on the
-[Chroma highlighter](https://github.com/alecthomas/chroma). {{% /alert %}}
+[Prism syntax highlighting](https://prismjs.com/download.html#themes=prism). {{% /alert %}}
 
 The list of languages supported by chroma can be found in their
 [GitHub repository](https://github.com/alecthomas/chroma#supported-languages).
```
Comment (gwarf) [inline content/en/about/contributing/syntax/_index.md:138]: If chroma is no more used this should be replaced/removed.

File: i18n/de.toml [hunk 11]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (gwarf) [inline i18n/de.toml:1]: Why would we need/want to include this file while german is not a supported language?
Same for the other language I don't think we should allow/enable in the conf and ship files for another language but English.
If there is a legit reason for doing this could you please share it?

File: layouts/partials/tags.html [hunk 12]
```diff
@@ -0,0 +1,7 @@
+{{ if .Params.tags }}
+<div class="tags">
+{{range .Params.tags}}
+  <a class="tag-link" href="{{ "/tags/" | relLangURL }}{{ . | urlize }}">{{ . }}</a>
+{{end}}
+</div>
+{{end}}
```
Comment (gwarf) [inline layouts/partials/tags.html:7]: Please always add a newline at end of files

File: layouts/partials/toc.html [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+{{ with .TableOfContents }}
+{{ if ge (len .) 200 }}
+<div class="td-toc">
+    <p class="td-toc-title">{{ T "on_this_page" }}</p>
+    <hr class="panel-line">
+    {{ . }}
+</div>
+{{ end }}
+{{ end }}
+{{ end }}
```
Comment (gwarf) [inline layouts/partials/toc.html:11]: Newline.

File: layouts/shortcodes/tabx.html [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+<!-- store all tab info in dict tab -->
+{{ $tab := dict "header" $header }}
+{{ with $.Inner }}
+{{ $tab = merge $tab (dict "content" $.Inner) }}
+{{ end }}
+
+<!-- add dict tab to parent's scratchpad -->
+{{ with .Parent }}
+{{- $.Parent.Scratch.SetInMap "tabs" (printf "%v" $.Ordinal) $tab -}}
+{{ end }}
```
Comment (gwarf) [inline layouts/shortcodes/tabx.html:20]: Newline

File: static/css/egi.css [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+  vertical-align: top;
+}
+
+.list-inline .github {
+  margin-top: 2px;
+}
+
+div.code-toolbar > .toolbar {
+	right: 0.45em;
+}
```
Comment (gwarf) [inline static/css/egi.css:134]: Newline

File: config.toml [hunk 16]
```diff
@@ -92,17 +106,22 @@ algolia_docsearch = false
...
 
-github_branch= "main"
+# Enable syntax highlighting and copy buttons on code blocks with Prism
+prism_syntax_highlighting = true
 
 # User interface configuration
 [params.ui]
 # Enable to show the side bar menu in its compact state.
 sidebar_menu_compact = true
+sidebar_menu_foldable = false
```
Comment (gwarf) [inline config.toml:119]: I've been wondering about this one, maybe it would be useful for use as like under `/users` we have lots of entries, it may make things a bit cleaner/more visible

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, that's quite a nice (and big) PR, super happy to see this being done!
I'added a few comments here and there, please have a look, thanks!

File: content/en/_index.md [hunk 17]
```diff
@@ -17,65 +17,73 @@ description: "Documentation related to EGI activities"
...
-{{% blocks/feature icon="fab fa-github" title="Contributions welcome!"
+
+{{% blocks/feature icon="fab fa-github" title="Contribute!"
     url="https://docs.egi.eu/about/contributing" %}}
+
 <!-- markdown-link-check-enable-->
-We do a [Pull Request](https://github.com/EGI-Federation/documentation/pulls)
-contributions workflow on **GitHub**. New users are always welcome!
+
+We do a contributions workflow on GitHub. New users are always welcome!
```
Comment (enolfc) [inline content/en/_index.md:76]: This reads weird. Like missing a verb maybe?

**Review by enolfc — COMMENTED** [review]
This is a big one. I mostly went through the md files and added few things.

File: config.toml (hunk 16 above)
Comment (thebe14) [reply config.toml]: There is a massive change coming on how sections are organized and how pages are written (also to make the left-side navigation tree happy and work optimally).

File: content/en/about/contributing/syntax/_index.md [hunk 18]
```diff
@@ -22,6 +22,42 @@ For references The following shortcodes are available:
...
+---
+title: "Syntax"
+description: "Syntax guide for EGI documentation"
+type: docs
+weight: 30
+---
+```
+
+The parameter `weight` controls the order of the chapters on the same level in
+the left-side navigation tree. Chapters will appear in descending order of their
```
[the remark this answers is not in the document]
Comment (thebe14) [reply content/en/about/contributing/syntax/_index.md:43]: It's ascending. Corrected in a subsequent commit.

File: content/en/_index.md (hunk 17 above)
Comment (thebe14) [reply content/en/_index.md:76]: It's what is on the site now. Updated.

File: static/css/egi.css (hunk 15 above)
Comment (thebe14) [reply static/css/egi.css:142]: Fixed.

File: layouts/shortcodes/tabx.html (hunk 14 above)
Comment (thebe14) [reply layouts/shortcodes/tabx.html:20]: Fixed.

File: layouts/partials/toc.html (hunk 13 above)
Comment (thebe14) [reply layouts/partials/toc.html:11]: Fixed

File: layouts/partials/tags.html (hunk 12 above)
Comment (thebe14) [reply layouts/partials/tags.html:7]: Fixed

File: config.toml (hunk 5 above)
Comment (thebe14) [reply config.toml:88]: Done

File: config.toml (hunk 4 above)
Comment (thebe14) [reply config.toml:67]: Fixed.

File: config.toml (hunk 3 above)
Comment (thebe14) [reply config.toml:41]: Fixed

File: config.toml (hunk 2 above)
Comment (thebe14) [reply config.toml:32]: Removed

File: config.toml [hunk 19]
```diff
@@ -116,46 +137,47 @@ enable = true
...
+[params.katex]
+enable = true
+html_dom_element = "document.body"
+
+# enable display of chemical equations and physical units via mhchem extension
+[params.katex.mhchem]
+enable = true
+
+[params.print]
+disable_toc = true
```
Comment (gwarf) [inline config.toml:183]: Missing newline.

File: config.toml [hunk 20]
```diff
@@ -92,16 +106,23 @@ algolia_docsearch = false
...
 
-github_branch= "main"
+# Enable syntax highlighting and copy buttons on code blocks with Prism
+prism_syntax_highlighting = true
 
 # User interface configuration
 [params.ui]
 # Enable to show the side bar menu in its compact state.
-sidebar_menu_compact = true
+sidebar_menu_compact = false
```
Comment (gwarf) [inline config.toml:118]: I tend to prefer having things compacted, and letting users open what they want/need and not confusing them with a too long list

File: config.toml (hunk 1 above)
Comment (thebe14) [reply config.toml:17]: I introduced several new translated text blobs. This just flags all untranslated strings with the placeholder [i18n] identifier. It is easier to see if there is non-content stuff that is not localized. Can be turned off, but what I added is ready for multilanguage support.

File: config.toml (hunk 19 above)
Comment (thebe14) [reply config.toml:180]: Fixed

File: config.toml (hunk 1 above)
Comment (gwarf) [reply config.toml:17]: But I don't think we plan/want any multi-language support. Would already be good if we could have proper up to data and maintained docs in English :)

File: config.toml (hunk 20 above)
Comment (thebe14) [reply config.toml:115]: This only keeps open the path from what you're reading to the root. It makes extremely hard to read documentation about a topics that is spread in sibling files (on the same level), eventually having to jump to one level up to a page that is not in the path to the root. This documentation site is not simple enough or organized well enough to make this kind of reading a good experience. Or at least this is my opinion so far. We can put it back, if you wish.

Comment (thebe14) [reply config.toml:115]: Not to mention, I had to test that when the left-side navigation tree does not fit on the page, it is placed and scrolls properly.

File: assets/scss/_variables_project.scss [hunk 21]
```diff
@@ -0,0 +1,19 @@
...
+
+// Dysplay styles
+$display1-size: 2.5rem !default;
+$display2-size: 2rem !default;
+$display3-size: 1.5rem !default;
+$display4-size: 1.12rem !default;
+
+// Space
+
+$spacer: 0.5rem;
```
Comment (gwarf) [inline assets/scss/_variables_project.scss]: I'm really not fond of this, I feel it's making text way to small/difficult to read, and while I know it's easy for people to use a custom zoom with a few keybindings (like I had to do), not all users are aware and something that's easy to read should be the default.
I would personally prefer to have something less small.
Curious to hear about others / @enolfc  about this one.

Comment (thebe14) [reply assets/scss/_variables_project.scss]: I have the exact opposite experience. It was so big on my screens, that I had to constantly use it at ~50%. To judge the scale, the Github buttons at the bottom come at a standard size, it is very hard to resize them, as the style is injected inline. Sites with those buttons look normal (meaning the site content and the size of those buttons is in harmony), whereas on our documentation site, it scrolled (out) to be usable, these buttons are almost unreadable. This to my says that the durrent documentation site is scaled wrong.

File: config.toml (hunk 1 above)
Comment (thebe14) [reply config.toml:17]: I know. But I learned to do things right :)

File: content/en/internal/_index.md [hunk 22]
```diff
@@ -6,6 +6,7 @@ type: "docs"
 menu:
   main:
     weight: 30
+    pre: <i class='fa fa-cogs'></i>
```
Comment (gwarf) [inline content/en/internal/_index.md:9]: I feel this is making things more complex/difficult to read, especially as the icon is really close to the text, I don't feel like it works well with the current setup:
Screenshot 2021-07-12 at 14 54 51

Comment (thebe14) [reply content/en/internal/_index.md:9]: It's easy to remove.

Comment (thebe14) [reply content/en/internal/_index.md:9]: But on my browsers looks better. 

Opera:
image

Edge (the new Chromium-based one):
image

Firefox:
image

Comment (gwarf) [reply content/en/internal/_index.md:9]: So the theme wasn't properly updated in my first checkout, no that's fine.
Yes it's a lot more clear, thanks!
Screenshot 2021-07-12 at 15 28 21

Let's keep it and see how it goes.

File: assets/scss/_variables_project.scss (hunk 21 above)
Comment (gwarf) [reply assets/scss/_variables_project.scss]: With the new size, at 100% it's barely readable for me with my old used eyes  on my monitor (3440x1440) and laptop screen (2880x1800).
I've got to zoom to at least 135% to be able to read the text easily enough.
While for the docs.egi.eu I happily use the default zoom level (no zoom). 

I would have expected that the theme default would be something quite generic with regards to size characters to fit most setup, and if it's not we should try to find what would make the more sense/be the most versatile.
Maybe the comms team may have some suggestions?
(Too bad we can't currently have a test instance of the PR to see how they look like without having to test locally, to let people easily check...)

File: config.toml (hunk 20 above)
Comment (gwarf) [reply config.toml:115]: OK fine.

File: assets/scss/_variables_project.scss (hunk 21 above)
Comment (gwarf) [reply assets/scss/_variables_project.scss]: So let's keep this font size and get feedback from colleagues.

File: i18n/de.toml (hunk 11 above)
Comment (gwarf) [reply i18n/de.toml]: The content file is included as part of the partial update, it was agreed to only keep the english one.

**Comment by gwarf** [comment]
> Running a HTML linter on a Hugo project is useless. The HTML files are just fragments, heavily sprinkled with Go blocks. The linter starts with insisting the <!doctype> tag is missing before it accepts any other tag, then just chokes on these files. Please turn it off, on this project its "insights" are garbage.

Please remove this directory from the HTML linter configuration, if possible.
If it's really not possible let's just disable it totally.

File: config.toml (hunk 1 above)
Comment (gwarf) [reply config.toml]: It was agreed to revert this.

File: content/en/about/contributing/syntax/_index.md (hunk 8 above)
Comment (thebe14) [reply content/en/about/contributing/syntax/_index.md:31]: OK, will update. For now I restored the current content.

File: content/en/about/contributing/syntax/_index.md (hunk 9 above)
Comment (thebe14) [reply content/en/about/contributing/syntax/_index.md:25]: OK, will update. For now I restored the current content.

File: content/en/about/contributing/syntax/_index.md (hunk 10 above)
Comment (thebe14) [reply content/en/about/contributing/syntax/_index.md:139]: Will replace it in the new PR. For now I restored the current content under this section.

**Comment by thebe14** [comment]
> > Running a HTML linter on a Hugo project is useless. The HTML files are just fragments, heavily sprinkled with Go blocks. The linter starts with insisting the <!doctype> tag is missing before it accepts any other tag, then just chokes on these files. Please turn it off, on this project its "insights" are garbage.
> 
> Please remove this directory from the HTML linter configuration, if possible.
> If it's really not possible let's just disable it totally.

Excluded

File: content/en/about/contributing/syntax/_index.md (hunk 9 above)
Comment (gwarf) [inline content/en/about/contributing/syntax/_index.md:25]: I thought you removed/renamed the parts about chapters?
Please replace by section for now.

File: content/en/about/contributing/syntax/_index.md [hunk 23]
```diff
@@ -22,6 +22,42 @@ For references The following shortcodes are available:
 - [Hugo's shortcodes](https://gohugo.io/content-management/shortcodes/)
 - [Docsy theme shortcodes](https://www.docsy.dev/docs/adding-content/shortcodes/)
 
+### Adding chapters
+
+To create new chapters in the documentation (visible in left-side navigation
```
Comment (gwarf) [inline content/en/about/contributing/syntax/_index.md:27]:
```suggestion
To create a new section in the documentation (visible in left-side navigation
```

File: content/en/about/contributing/syntax/_index.md [hunk 24]
```diff
@@ -22,6 +22,42 @@ For references The following shortcodes are available:
...
+tree), you have to create new folders under `/content/<language>` and add a file
```
Comment (gwarf) [inline content/en/about/contributing/syntax/_index.md:28]:
```suggestion
tree), you have to create a new folder under `/content/en` and add a file
```

File: content/en/about/contributing/syntax/_index.md (hunk 8 above)
Comment (gwarf) [inline content/en/about/contributing/syntax/_index.md:31]:
```suggestion
[front matter](https://gohugo.io/content-management/front-matter/) in YAML
holding the metadata of the page:
```

File: content/en/about/contributing/syntax/_index.md [hunk 25]
```diff
@@ -58,7 +94,8 @@ The following code:
 
 Will render as:
 
-{{% pageinfo %}} This is a placeholder. {{% /pageinfo %}}
+{{% pageinfo %}} This is a
+placeholder. {{% /pageinfo %}}
```
Comment (gwarf) [inline content/en/about/contributing/syntax/_index.md:98]:
```suggestion
{{% pageinfo %}} This is a placeholder. {{% /pageinfo %}}
```

File: content/en/about/contributing/syntax/_index.md (hunk 9 above)
Comment (gwarf) [reply content/en/about/contributing/syntax/_index.md:25]:
```suggestion
### Adding sections
```

File: layouts/partials/footer.html [hunk 26]
```diff
@@ -0,0 +1,59 @@
...
+        {{ end }}
+        {{ end }}
+      </div>
+      <div class="col-12 col-sm-4 text-center py-2 order-sm-2">
+        {{ with .Site.Params.copyright }}<small class="text-white">&copy; {{ now.Year}} {{ .}} {{ T "footer_all_rights_reserved" }}</small>{{ end }}<br/>
+        {{ with .Site.Params.privacy_policy }}<small class="ml-1"><a href="{{ . }}" target="_blank" rel="noopener">{{ T "footer_privacy_policy" }}</a></small>{{ end }}<br/>
+	{{ if not .Site.Params.ui.footer_about_disable }}
+		{{ with .Site.GetPage "about" }}<small><a href="{{ .RelPermalink }}">{{ .Description }}</a></small>{{ end }}
+	{{ end }}
+      </div>
```
Comment (gwarf) [inline layouts/partials/footer.html:25]: Indentation seems weird

File: layouts/partials/navbar.html [hunk 27]
```diff
@@ -0,0 +1,37 @@
+{{ $cover := .HasShortcode "blocks/cover" }}
+<nav class="js-navbar-scroll navbar navbar-expand navbar-dark {{ if $cover}} td-navbar-cover {{ end }}flex-column flex-md-row td-navbar">
+        <a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
+		<span class="navbar-logo">{{ if .Site.Params.ui.navbar_logo }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span><span class="text-uppercase font-weight-bold">{{ .Site.Title }}</span>
+	</a>
```
Comment (gwarf) [inline layouts/partials/navbar.html:5]: Here it's indented with many spaces while in footer.html it's only two spaces, better to be coherent and use 2 spaces everywhere

Comment (gwarf) [reply layouts/partials/navbar.html:5]: (same for other partials)

File: static/css/egi.css [hunk 28]
```diff
@@ -0,0 +1,142 @@
+.features {
```
Comment (gwarf) [inline static/css/egi.css:1]: It would be good to have some header/comments explaining why we have all this customised.

File: static/js/egi.js [hunk 29]
```diff
@@ -0,0 +1,288 @@
+jQuery(document).ready(function () {
```
Comment (gwarf) [inline static/js/egi.js:1]: I saw that you disabled the check of this file, but you didn't fixed any of the problems like at least the inconsistent indentation, it would already have been better to have some default rules enforced instead of nothing at all.
Please also add an header/comments to clarify why we have this/the purpose

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks, I've added a few suggestions and comments, please address them so that we can finally merge this big PR!

File: assets/scss/_variables_project.scss (hunk 21 above)
Comment (enolfc) [reply assets/scss/_variables_project.scss]: I agree with @gwarf here, this is too small to read comfortably. I am going for 125% zoom to get a good experiece
Also, lines become too long to read easily:
Screenshot 2021-07-13 at 10 47 30

Comment (enolfc) [reply assets/scss/_variables_project.scss]: ok, so after discussing with @thebe14, I think we can move on

**Review by enolfc — COMMENTED** [review]
Good to go with the changes suggested by @gwarf. Not approving directly because the approval will be removed if you commit any changes, but will do as soon as these are in place

File: assets/scss/_variables_project.scss (hunk 21 above)
Comment (gwarf) [reply assets/scss/_variables_project.scss]: I think that we can commit like this to collect feedback from colleagues and decide if we need to increase the font.

File: content/en/about/contributing/syntax/_index.md (hunk 9 above)
Comment (thebe14) [reply content/en/about/contributing/syntax/_index.md:25]: I did, but then stashed content-related changes while on call with you.

File: layouts/partials/navbar.html (hunk 27 above)
Comment (thebe14) [reply layouts/partials/navbar.html:5]: Agree

File: layouts/partials/footer.html (hunk 26 above)
Comment (thebe14) [reply layouts/partials/footer.html:25]: Fixed

File: content/en/about/contributing/syntax/_index.md (hunk 25 above)
Comment (thebe14) [reply content/en/about/contributing/syntax/_index.md:98]: This render weird in some places. I discovered that adding a line break will render it aligned in the middle vertically.

File: layouts/partials/navbar.html (hunk 27 above)
Comment (thebe14) [reply layouts/partials/navbar.html:5]: This is how I took them from the theme. Auto-formatting this Go-sprinkled html makes it really bad, sometimes breaks stuff.

File: static/js/egi.js (hunk 29 above)
Comment (thebe14) [reply static/js/egi.js:1]: I disagree with some of the rules being enforced (e.g. no ; at the end of statements). 
I did an auto-format on it, but did not like the results. I may have to work on the rules for JavaScript. 

But the biggest problem is that it seems that this is being checked as it was an independent file, although this gets injected by some Go code into a HTML fragment (the page's head) after e.g. jQuery was already included. The linter flags a massive amount of errors on anything that starts with $.

Comment (thebe14) [reply static/js/egi.js:1]: Added header

File: static/css/egi.css (hunk 28 above)
Comment (thebe14) [reply static/css/egi.css:1]: Added

**Review by enolfc — APPROVED** [review]

**Review by gwarf — APPROVED** [review]
Finally approving, thanks!
